src/main/java/com/minecolonies/api/items/registry/ToolTypeEntry.java

```diff
@@ -0,0 +1,198 @@
+package com.minecolonies.api.items.registry;
+
+import com.minecolonies.api.items.ModToolTypes;
+import com.minecolonies.api.util.constant.IToolType;
+import net.minecraft.network.chat.Component;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.registries.RegistryObject;
+
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+/**
+ * An entry in the ToolType registry that defines the types of
+ * tools within the colony.
+ */
+public class ToolTypeEntry implements IToolType
+{
+    /**
+     * The name of the tool type.
+     */
+    private final String name;
+
+    /**
+     * The component for the human readable name.
+     */
+    private final Component displayName;
+
+    /**
+     * Predicate to determine whether a given ItemStack
+     * can act as this tool type.
+     */
+    private final Predicate<ItemStack> isTool;
+
+    /**
+     * A function to return the integer item level of a
+     * given ItemStack.
+     */
+    private final Function<ItemStack, Integer> itemLevel;
+
+    /**
+     * Constructor.
+     *
+     * @param name        The name of the tool type
+     * @param displayName The human readable name of the tool type
+     * @param isTool      A predicate for determining if an itemstack is the tool type
+     * @param itemLevel   A function to return the item level of an item stack
+     */
+    private ToolTypeEntry(final String name, final Component displayName, final Predicate<ItemStack> isTool, final Function<ItemStack, Integer> itemLevel)
+    {
+        this.name = name;
+        this.displayName = displayName;
+        this.isTool = isTool;
+        this.itemLevel = itemLevel;
+    }
+
+    /**
+     * Get the registered ToolTypeEntry from a tool ID.
+     *
+     * @param toolId The target ID
+     * @return The ToolTypeEntry
+     */
+    public static IToolType getToolType(final String toolId)
```

Comment:
Remove method and use the API proxy to get the tool types.

Reply:
Done.